openstack/swift/aggregations/kubernetes/consolidation.rules

```diff
@@ -0,0 +1,5 @@
+groups:
+- name: swift
+  rules:
+    - record: swift_container_memory_usage_percent
+      expr: '100 * max by (namespace, pod_name) (container_memory_working_set_bytes{pod_name=~"swift-.+", container_name!="POD"}) / max by (namespace, pod_name) (label_replace(kube_pod_container_resource_limits_memory_bytes{pod=~"swift-.+"}, "pod_name", "$1", "pod", "(.*)"))'
```

Comment:
The `label_replace` should not be necessary anymore. The label sets of those metrics have been consolidated. @auhlig knows more.